app/src/main/java/org/mozilla/fenix/trackingprotection/TrackingProtectionOverlay.kt

```diff
@@ -49,7 +49,7 @@ class TrackingProtectionOverlay(
 
     @Suppress("MagicNumber", "InflateParams")
     private fun showTrackingProtectionOnboarding() {
-        if (!getToolbar().hasWindowFocus()) return
+        if (getToolbar().translationY > 0) return
```

Comment:
Should we also keep the `getToolbar().hasWindowFocus()` check?

Reply:
I didn't notice any change with or without the check, but I'll include it in the next commit.